scripts/_rds.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+function _rds_help() {
+    echo
+    echo "uhd rds <command> [options]"
+    echo
+    echo "commands:"
+    echo "  help                                          - this help screen"
+    echo 
+    echo "  remove-deletion-protection <cluster-id>       - removes deletion protection and allows the cluster to be deleted"
```

Comment:
I left this in as a utility command in case anyone needs it, but it's not directly being used in the github action workflows